.github/workflows/canary-integration-test.yml

```diff
@@ -47,6 +47,11 @@ jobs:
         GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
         docker images
         docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+    
+    - name: validate yaml 
+      run: |
+        cd cluster/examples/kubernetes/ceph/
+        kubectl create  $(ls -I scc.yaml -I operator-openshift.yaml -I "*.sh" -I "*.py" -p | grep -v / | awk ' { print " -f " $1 } ') --dry-run
```

Comment:
How about a comment about skipping some yamls that are only for openshift?

Reply:
Does this command succeed for the CRs even though the CRDs and namespace hasn't been created? I wonder if we would need to create the crds.yaml and common.yaml before we can create the CRs.

Reply:
Right, we do need to create them. 
Done.